generator/src/test/java/com/scottlogic/deg/generator/cucumber/MatchingRegex.feature

```diff
@@ -128,29 +128,13 @@ Scenario: Running a 'matchingRegex' request that includes special characters (no
        | "け" |
        | "げ" |
 
-@ignore #issue 294
 Scenario: Running a 'matchingRegex' request that includes special characters (emoji) only should be successful
      Given foo is matching regex /[😁-😘]{1}/
      Then the following data should be generated:
       | foo  |
       | null |
-      | "😁" |
-      | "😂" |
-      | "😃" |
-      | "😄" |
-      | "😅" |
-      | "😆" |
-      | "😉" |
-      | "😊" |
-      | "😋" |
-      | "😌" |
-      | "😍" |
-      | "😏" |
-      | "😒" |
-      | "😓" |
-      | "😔" |
-      | "😖" |
-      | "😘" |
+      | null |
```

Comment:
Why does this generate 3 nulls? Is that desired?

Reply:
Agreed, this shouldn't be happening, null should.only be emitted once.

Reply:
We need to understand why it produces duplicates as opposed to outright removing them after it's produced - thinning for duplicates after the fact would introduce additional overhead